strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/JwtAuthenticationTest.java

```diff
@@ -141,6 +143,27 @@ public void testJWTExpirationToken()
                .body("error", equalTo("expired"));
     }
 
+    @Test
+    public void testJWTIssuedAtFuture()
+            throws Exception
+    {
+        String url = getContextBaseUrl() + "/users";
+
+        NumericDate futureNumericDate = NumericDate.now();
+        // add five minutes to the current time to generate a JWT issued in the future
+        futureNumericDate.addSeconds(300);
+
+        String token = securityTokenProvider.getToken("admin", Collections.emptyMap(), 10, futureNumericDate);
+
+        given().header(HttpHeaders.AUTHORIZATION, getAuthorizationHeader(token))
+                .header(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .get(url)
+                .then()
+                .statusCode(HttpStatus.UNAUTHORIZED.value())
+                .body("error", equalTo("invalid.token"));
```

Comment:
Cosmetic: Could you please align these by the dot, in accordance with our 
[coding convention](https://strongbox.github.io/developer-guide/coding-convention.html)? Thanks! :)